docker-compose/Makefile

```diff
@@ -18,7 +18,8 @@ endif
 DOCKER_HOST_IP ?= $(shell echo ${DOCKER_HOST} | grep -o "[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}" || echo ${LOCAL_IP})
 DOCKER_REGISTRY ?= ""
 DOCKER_IMAGE_PREFIX ?= openwhisk
-OPENWHISK_PROJECT_HOME ?= ./openwhisk-master
+OPENWHISK_VERSION ?= master
```

Comment:
Let’s expose this option in the README as well, to make it easy for developers to run the latest official release, ideally with links to the releases page, to quickly find the latest version. WDYT ?